src/App.js

```diff
@@ -1,13 +1,51 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm.js';
+import TodoList from './components/TodoComponents/TodoList.js'
+import Todo from './components/TodoComponents/Todo.js'
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
```

Comment:
In order to add unique tasks, you may need to create an array of objects in this state.